CHANGELOG.md

```diff
@@ -22,6 +22,7 @@ This project adheres to [Semantic Versioning](https://semver.org/).
 - Drop support for PHP 7.2 (#1111)
 
 ### Fixed
+- Bump the minimum Symfony 4.4. version (#1187)
```

Comment:
It took me a bit of time to understand this.  I think "4.4.x" is much clearer than "4.4." which looks like the end of a sentence.  Also we should give the reason (such as "to avoid a PHP deprecation warning").

Reply:
Ah, no, the additional period was just a typo. :-)

I'll update the changelog entry in a second.

Reply:
I've fixed it in the commit title too.